config/locales/en.yml

```diff
@@ -53,6 +54,13 @@ en:
       student_loan_amount: "How much student loan did you repay while you were at %{claim_school_name} between 6 April XXXX and 5 April XXXX?"
       email_address: "What is your email address?"
       bank_details: "Enter bank account details"
+      eligible_subjects:
+        biology_taught: "Biology"
+        chemistry_taught: "Chemistry"
+        physics_taught: "Physics"
+        computer_science_taught: "Computer Science"
+        languages_taught: "Languages"
+        not_applicable: "Not applicable, I didn't teach any of these subjects"
```

Comment:
As mentioned on the last PR, "not applicable" doesn't feel like a valid answer to the question (it would imply that the user thinks the question isn't relevant to them). Shouldn't this be "No, I didn't teach any of these subjects", or just "I didn't teach any of these subjects"?

Reply:
I'm going on what was agreed in the card. I reckon we should stick with what's been agreed and see what comes out of testing 👍

Reply:
Fair enough 👍